Dockerfile

```diff
@@ -15,11 +14,12 @@ WORKDIR /tmp
 # install baseline system packages
 RUN apt-get update \
     && apt-get install -y \
-    bash ca-certificates curl git jq openssh-client openssl python sudo tar ncurses-base
+    bash ca-certificates curl git groff jq openssh-client openssl python sudo tar ncurses-base
 
-# install serverless framework
-RUN apt-get install -y nodejs npm \
-    && npm install -g serverless@${SERVERLESS_VERSION}
+# install node and serverless framework
+RUN curl -sL "https://deb.nodesource.com/setup_${NODE_VERSION}.x" | bash - \
+    && apt-get install -y nodejs \
+    && sudo npm install -g serverless
```

Comment:
it's nice to be able to hard core serverless when there are breaking changes. this happened to us on firewall.

Reply:
I have added the version we're on here - not going to push another image but it'll be in the repo for reference